mimic/rest/auth_api.py

```diff
@@ -190,6 +191,46 @@ def validate_token(self, request, token_id):
             response["access"]["RAX-AUTH:impersonator"] = impersonator_user_role(
```

Comment:
Yes, we the token being validated was got from impersonation, we expect the impersonator role to exist.